test/e2e/apimachinery/crd_publish_openapi.go

```diff
@@ -683,10 +684,15 @@ func convertJSONSchemaProps(in []byte, out *spec.Schema) error {
 	if err := apiextensionsv1.Convert_v1_JSONSchemaProps_To_apiextensions_JSONSchemaProps(&external, &internal, nil); err != nil {
 		return err
 	}
-	if err := validation.ConvertJSONSchemaPropsWithPostProcess(&internal, out, validation.StripUnsupportedFormatsPostProcess); err != nil {
+	kubeOut := kubeopenapispec.Schema{}
+	if err := validation.ConvertJSONSchemaPropsWithPostProcess(&internal, &kubeOut, validation.StripUnsupportedFormatsPostProcess); err != nil {
```

Comment:
Can we remove the need for stripping unsupported formats since we now have control over the formats from strfmt?

Reply:
I think so

Reply:
Am not convinced thought. It is another safety net without value to remove it.